web-console/knikubevirt/import-vm/import-vm.md

```diff
@@ -0,0 +1,71 @@
+
+![Import VM](img/Import-vm.png)
+
+The Import virtual machine flow is very much the same as the Create a virtual machine flow with the exception of the general step.
+
+![General step](img/Step-1-basic-import-0.png)
+
+## Selecting provider 
+The Provider is chosen (in this case VMware is the only one available).
+
+![provider](img/Step-1-basic-import-1.png)
+
+## Selecting vCenter instance
+Once VMware is selected a new `vCenter instance` field appears. The user clicks it to see a list of previously-connected vCenter instances that have been stored as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/). The user selects one.
+
+![vCenter provider](img/Step-1-basic-import-2.png)
+
+## Connecting to new instance
+The user can also connect to a new vCenter instance by clicking `Connect to new instance` from within the dropdown.
+
+![connecting to new instance](img/Step-1-basic-import-5.png)
+
+New fields for vCenter URL, vCenter Username, vCenter Password, and VM to import appear along with a checkbox to `Save as new vCenter instance secret`. This would be creating a new secret so it would be unchecked by default.
+
+The vCenter URL field should include a syntax hint with the desired formatting. The user is likely to paste a variety of URLs, so any FQDN the user submits should be validated and automatically corrected before being used to communicate with vCenter’s API.
+
+![check connection](img/Step-1-basic-import-6.png)
+
+When the user clicks the “Check” button a spinner icon appears within the button to indicate that the credentials are being checked.
+
+![checking connection](img/Step-1-basic-import-7.png)
+
+
+## Selecting virtual machine
+When a vCenter instance is selected, The `Virtual Machine to import` dropdown is available with all the available VMs.
+
+![selected virtual machine](img/Step-1-basic-import-3.png)
+
+## vCenter connection error
+If the wizard cannot connect to the vCenter instance using the credentials stored within its Secret, an error message will be displayed. If the credentials are incorrect they can update them. If they check `update vCenter credentials secret` the action will be `check and update`. This should be the default because they have already created this secret. This allows them to update the secret without having to leave this flow.
+
+![Updating vCenter credentials](img/Step-1-basic-import-4.png)
+
+![checking vCenter credentials](img/Step-1-basic-import-4.2.png)
+
+Once the credentials are checked and updated the username/password fields collapse.
+
+![collapse vCenter credentials](img/Step-1-basic-import-4.3.png)
+
+If the user un-checks the checkbox the action label will become `Check`.
+
+![check vCenter credentials](img/Step-1-basic-import-4.1.png)
+
+### Save credentials 
+The `Save as new vCenter instance secret` checkbox becomes enabled to allow the user to save them as a new Kubernetes Secret.
+
+![save secret](img/Step-1-basic-import-7.1.png)
```

Comment:
This one image still uses "Review and create" for the secondary action button rather than the new "Review and import". The rest all look updated. 👍